users.json

```
@@ -48,5 +43,10 @@
     "username": "ambrogio-bot",
     "role": "admin",
     "publicKey": "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIJtnZ3DbWbrDAMJ9lmy/8Ny6DHNNTk31ec9X3wEXkj/9"
+  },
+  {
+    "username": "ale",
+    "role": "admin",
```

```suggestion
    "username": "aleclearmind",
    "role": "member",
```

Sorry if I didn't notice before but could you please add yourself as `member` and also use your GitHub username, this way @ambrogio-bot will automatically add you to the organization once the PR is merged (actually you'll receive an invite by e-mail that you will need to accept).

---

I tried to trick you to see if you would notice the change `:D`
Anyway, I switched to my GitHub username.

---

Lol :rofl: 
It's not that easy to trick me (`proceeds to destroy the terraform state inadvertently`...)